stylesheet.css

```diff
@@ -0,0 +1,101 @@
+*{
+    box-sizing:border-box;
+}
+
+body {
+    background-color :#f2f2f2;
+    font-size: 62.5%;
+    font-family: HKGrotesk, sans-serif;
+    height: 100vh;
+    width: 100vw;
+    position: relative;
+}
+.hometop {
+    background-color: #F8D5E2;
+    height: 50vh;
+    width: 100vw;
+    position:relative;
+    border-top: 5px solid #f2f2f2;
+    border-left: 5px solid #f2f2f2;
+    border-right: 5px solid #f2f2f2;
+
+}
+nav {
+    width: 98%;
+    padding: 10px;
+}
+.mypicture {
+    width: 100px;
+    height: 100px;
+    border: 2px solid cornflowerblue;
+    border-radius: 50%;
+    margin: 0;
+    padding: 0;
+}
+nav a {
+    text-decoration: none;
+    font-size: 1rem;
+    color: black;
+    float: right;
+    margin: 14px;
+}
+a:hover {
+    color: cornflowerblue;
+}
+
+.homecenter {
+    /*margin: 0 auto;*/
+    width: 100%;
+    height: 90%;
+    /*display: flex;*/
+    /*justify-content: center;*/
+    /*align-content: center;*/
+    position: absolute;
+    top: 5%;
+    z-index: 2;
+}
+.arrows {
```

Comment:
i think if you get rif of the height the arrows will change the position completely